Documentation/General/finalReport/outlook.tex

```diff
@@ -1,2 +1,40 @@
 \chapter{Outlook}
-%the possible improvements in the future and the strategy to achieve these improvements
\ No newline at end of file
+During the project, we developed several improvements to the system.
+These improvements could not have been implemented due to time constraints, but the system designed in such a way that support for these features can be easily achieved.
+
+\begin{enumerate}
+\item Our initial selling point was a neural-network based suggestion system.
+We had to drop this sub-system due to lack of initial data, but our current system is able to (over time) provide this data.
+Therefore in the future, the data is available to train the neural network and predict valence and arousal values of a song when it is inserted in the database.
```

Comment:
Add arousal and valence to glossery, because this report should be readable on it's own.